book.rb

```diff
@@ -0,0 +1,11 @@
+# this class represents the books
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [ ] The rentals should be set for getter and setter since we will also be adding and reading it from the UI.

Reply:
Thank you @Olamarx. Done!